alfresco-search/src/test/java/org/alfresco/solr/AbstractAlfrescoSolrTests.java

```diff
@@ -18,16 +18,15 @@
  */
 package org.alfresco.solr;
 
+import static java.util.Arrays.stream;
+import static java.util.Optional.ofNullable;
+import static org.junit.Assert.assertEquals;
+
 import java.io.IOException;
 import java.math.BigInteger;
 import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Date;
-import java.util.List;
-import java.util.Locale;
-import java.util.Properties;
+import java.util.*;
```

Comment:
We typically don't use wildcard imports.

Reply:
I finally got where this thing needs to be set in Intellij so we should no longer have this problem